src/python/pants/init/engine_initializer.py

```diff
@@ -203,25 +204,28 @@ def run_console_rules(self, options_bootstrapper, goals, target_roots):
 
     :returns: An exit code.
     """
+
+    async_reporter = AsyncWorkunitHandler(self.scheduler_session, callback=None)
```

Comment:
This might not be the right scope for this. Should this have the same lifetime as the `RunTracker`?

Reply:
Yeah, I moved this to `local_pants_runner.py`, which means that `LocalPantsRunner` is now responsible for turning on streaming workunit reporting at the graph session init level, and instantiating this class. @benjyw thought that was a reasonable place to put this code.